jujugui/static/gui/src/app/state/state.js

```diff
@@ -274,6 +292,7 @@ const State = class State {
       }
     }
     next();
+    return matchingDispatchers.key;
```

Comment:
Here would simply return `key` as it would have returned with `undefined` above if there was no dispatchers found.